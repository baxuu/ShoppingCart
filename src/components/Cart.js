import React from 'react';

import ProductDetails from './productDetails/ProductDetails';
import ShippingCost from './shippingCost/ShippingCost';
import CartSummary from './cartSummary/CartSummary';
import CartHeader from './cartHeader/CartHeader';

import './Cart.scss';

const Cart = () => {
  return (
    <div className='Cart'>
      <CartHeader />
      <div className='cart__container'>
        <ProductDetails />
        <div className='cart__totals'>
          <ShippingCost />
          <CartSummary />
        </div>
      </div>
    </div>
  );
};

export default Cart;
