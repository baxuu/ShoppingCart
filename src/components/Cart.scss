.cart__container {
  display: flex;
  justify-content: space-between;
}
.cart__totals {
  display: flex;
  flex-direction: column;
}

@media (max-width: 991px) {
  .cart__container {
    display: block;
  }
  .cart__totals {
    align-items: flex-end;
  }
}
@media (max-width: 767px) {
  .cart__totals {
    width: 600px;
    margin: 0 auto;
  }
}
@media (max-width: 630px) {
  .cart__totals {
    width: 500px;
  }
}
@media (max-width: 533px) {
  .cart__totals {
    width: 400px;
  }
}
@media (max-width: 431px) {
  .cart__totals {
    width: 300px;
  }
}
