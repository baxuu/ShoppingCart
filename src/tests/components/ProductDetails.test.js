import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Cart from '../../components/Cart';

Enzyme.configure({ adapter: new Adapter() });

describe('Cart', () => {
  it('snapshot test', () => {
    const CartWrapper = shallow(<Cart />);
    expect(CartWrapper).toMatchSnapshot();
  });
});
