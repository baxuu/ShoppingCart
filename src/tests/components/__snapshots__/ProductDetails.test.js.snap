// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart snapshot test 1`] = `ShallowWrapper {}`;
